_targets/build.project.armv7m7-imxrt106x

```diff
@@ -96,8 +96,21 @@ b_mkscript_user() {
 			esac;
 			printf "%s\n" "$cmd"
 		done
+
+		if [[ -z "${TARGET_PROJECT}" ]] && [[ -d "${PREFIX_ROOTFS}"userbin ]]; then
```

Comment:
`[` is enough

Reply:
I don't understand the reason of introducing this variable. We have a bunch of redundant variables, defines and types. Many of them are used only one time.